src/ontology/edits/oeo-shared.omn

```diff
@@ -1253,6 +1253,30 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/840",
         OEO_00000350
     
     
+Class: <http://openenergy-platform.org/ontology/oeo/oeo-shared/decrease>
```

Comment:
All your implemented classes and relations should have an IRI that starts with OEO:0034, where 0034 identifies you as @nelekoehler. Did you set your numerical identifiers right? https://github.com/OpenEnergyPlatform/ontology/wiki/Numerical-Identifiers
The IRI should be added automatically, once the settings are right. Then it should look like this:
```suggestion
Class: <http://openenergy-platform.org/ontology/oeo/oeo-shared/OEO:0034xxxx>
```

Reply:
Ah yes, thanks, there was a mistake with my settings, it somehow changed to custom label, when I last opened protege. I'll change it.

Reply:
Same happend here #1639. Please check all your former PRs for more cases. You can fix them in this PR directly.